scripts/sync_makefiles.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -uo pipefail
+
+# GITHUB_TOKEN required scope: repo.repo_public
+
+git_mail="prometheus-team@googlegroups.com"
+git_user="prombot"
+branch="makefile_common"
+commit_msg="makefile: update Makefile.common with newer version"
+pr_title="Synchronize Makefile.common from prometheus/prometheus"
+pr_msg="Propagating changes from master Makefile.common located in prometheus/prometheus."
+org="prometheus"
+
+if [ -z "${GITHUB_TOKEN}" ]; then
```

Comment:
If `GITHUB_TOKEN` isn't set, the script will crash here with `GITHUB_TOKEN: unbound variable`. We could set the variable like this above:

```bash
GITHUB_TOKEN="${GITHUB_TOKEN:-}"
```

Reply:
Right, I forgot that `set -o` is doing this.